src/templates/auth_manifests.yaml.j2

```diff
@@ -397,3 +398,58 @@ rules:
     - '*'
   resources:
     - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: webhook-role
```

Comment:
same as the above comment on the name

Reply:
I won't replicate the comment, but please modify all the names

Reply:
Sure thanks for the catch. I ommited the `-kubeflow` suffix from those names since I don't think it serves a purpose in our case. My understanding is that upstream manifests use that in order to distinguish between roles from using seldon inside kubeflow and as a standalone deployment.

Reply:
Do you think I should also include the `-kubeflow` suffix and keep the name exactly as it is from the built manifests?

Reply:
yes I would add the `-kubeflow` since it will make it easier to compare to upstream and I don't see any harm with adding it.

Reply:
reiterating, there is in some of them